docs/manage/deletion-requests.md

```diff
@@ -25,8 +25,8 @@ Key features:
 
 ## Create a deletion request
 
-:::warning
-Deletion requests result in the permanent deletion of data. Once deleted, the data cannot be recovered. Ensure that you have appropriately backed up any necessary data before submitting a deletion request.
+:::caution
+After data deletion request is approved, data will be deleted from the customer org and no users within the customer org (admin or otherwise) will be able to access the data. Reiterating the data deletion requests should not be raised frivolously and any data deletion approvals should be given sufficient consideration.
 :::
```

Comment:
@JV0812, I suggest changing "caution" since it renders as "Info". There is no separate rendering for "caution" (see the [style guide](https://help.sumologic.com/docs/contributing/style-guide/#admonitions)).